### ISSUE TITLE
[EN][Spell][Unstable Affliction] warlock

### ISSUE THREAD

**Issue by Illusie** [Atlantiss/BugTracker#4267]
http://www.wowhead.com/spell=30108/unstable-affliction
http://cata.openwow.com/spell=30108

I just wonder to myself If the penalty for dispelling Unstable Affliction work as it should be?
** I don't know ** how it should be I'm just asking because something doesn't make sense to me.

I'll expain.
I did a Battleground which my Unstable Affliction got 42 times dispelles
Just for the illustration, Lets say the penalty is 15000 damage done to the dispeller. 

42*15000 = 630000 (630k) It means 42 times dispell should make this <- amount of damage.

But here we go and the reason I open This issue.
My Recount addon shows me difference numbers for the 42 times that Unstable Affliction dispelled The damage it does was only 62000! [62000/42=1500] only 1500 damage done to dispeller! [Average]

Now when I looked at the statistics I saw that like 27 of 42 dispelled UA was under (Absorbs)
The rest (like 15) was under (Immunity) Which i don't know what it means Immunity.
the dispeller is Immune to get the penalty for dispelling? How that can be?

*** I'm not talking about using Cloak of Shadow or Ice Block, Divine Shield which as far as I know they are not under catagory of "dispelling". (Yes it's make my Dots go away, but it's not acutally "Dispell") ***

Dispel - http://wowwiki.wikia.com/wiki/Dispel

Here is another Battleground I did. I got 13 times dispelled on UA - http://i.imgur.com/SHz5XfL.jpg
11 of 13 (Absorb)
1 of 13 (Immunity)
1 of 13 (Hit)

1 of 13 actually did damage to the dispeller, 12 Did not.

So my question is... If Priest or Mage or IDK who can dispell my Unstable Affliction has a shield on himself and dispell my UA. does the UA still damage him through shield?

If a Priest put shield with 26000 absorb and dispell my UA [which lets say suppost to hit him for 22k]
His shield absorb go to 4000? Or it still on 26000? Which maybe will make sense why few of my UA dispelled are under (Absorbs).

Or maybe the damage suppost to bypass the absorption and hit him directly?
Which i believe is not, because the damage is Shadow. but I am still asking.

If Shaman use Shamanstic Rage (Glyphed) which dispelling my UA does he get the penalty?
Why do I ask about shaman you may ask? Because he still get the silence effect. but he get the damage penalty too?

Bottom line:
I would like to know how exactly the Unstable Affliction works. and if it works fine as Retail.
And if the damage is correct. how the Penalty for dispelling work and every info you can give me for that Skill.

Waiting for answers, Thanks.
Illusie.

P.S
my first issue I open, so be easy with me.
by the way how do I label/tag Warlock - Class or whatever here?

**Comment by Helbringer** [comment]
Wellcome to the bugtracker! "by the way how do I label/tag Warlock - Class or whatever here?"
you dont, the bugtracker moderators/developers will do that for you once the report is considered worthy.

**Comment by krojo** [comment]
This Immune is from Player who have more then 3 time silen 
1 silen = 100%
2silen = 50% 
3 silen = 30%
4 or more silen = Immune 
Or something like that. 
So this Immune is from silen not from damage.

**Comment by Kejmil** [comment]
holy pal popping aura mastery+conc aura and dispelling?

**Comment by Illusie** [comment]
@krojo i missunderstand you.
look again at the screenshot i posted.
1 of 13 succeed and damaged the dispeller, 12 immune/absorb.

It doesn't make sense to me. if 13 UA dispelled.
the damage done to dispeller should be around 15k*13 = 195k damage done. on Recount the damage says 21k only.

I still waiting for some Scripter / Mod / Idk who, that will explain me how UA should work.
thank you.

**Comment by Illusie** [comment]
@Wolffenstein @Kejmil @armano2 what going on with that issue?

**Comment by Illusie** [comment]
bump

**Comment by Jenovas** [comment]
Just did a test today + played a few BG's vs Affli Lock and Unstable Affliction worked normally.
Closing.

If you find the bug again and find out what causes it + 100% sure it's a bug please open a new ticket.
